maxim/logger/components/types.py

```
@@ -26,15 +27,37 @@ class Entity(Enum):
 
 
-class DateTimeEncoder(json.JSONEncoder):
-    """DateTime encoder.
+class CustomEncoder(json.JSONEncoder):
+    """Custom encoder.
 
-    This class represents a date time encoder.
+    This class represents a custom encoder that can handle any object type.
     """
 
     def default(self, o):
```

Ruff (0.12.2)

Missing return type annotation for public function `default`

(ANN201)